packages/block-library/src/navigation-menu-item/editor.scss

```diff
@@ -82,9 +82,11 @@
 
 	&.is-editing,
 	&.is-selected {
-		box-shadow: 0 0 1px $light-gray-900 inset;
-		border-radius: 4px;
-		min-width: 30px;
+		min-width: 20px;
+	}
+
+	.block-editor-rich-text__editable.is-selected:not(.keep-placeholder-on-focus):not(:focus) [data-rich-text-placeholder]::after {
```

Comment:
That is quite the selector!

Reply:
proud to got it!

Reply:
What is this doing? I don't see any difference when I remove this rule. It sounds like something should be fixed for rich text instead of in the block.

Reply:
That seems to be correct, @ellatrix. The component has been developing constantly lately, and I think it's why these changes don't impact its behavior currently.

Reply:
I've removed it in #19822.

Reply:
Thanks. 🙏

Reply:
Actually, @ellatrix, it seems that we've introduced a regression.

![no-cartet-issue](https://user-images.githubusercontent.com/77539/72998391-8283da00-3ddc-11ea-8f0c-fa91a9583fc7.gif)

When the user tries to set the link text, it doesn't show the caret.

Reply:
Strange, I see it:

<img width="174" alt="Screenshot 2020-01-23 at 17 44 41" src="https://user-images.githubusercontent.com/4710635/73004708-0e5f2b80-3e08-11ea-8eac-83efa47380f5.png">

Is it caused by this PR?

Reply:
Yes. :-D

Fixed here: https://github.com/WordPress/gutenberg/pull/20075